app/experimenter/nimbus-ui/src/components/PageRequestReview/FormApproveConfirm.tsx

```diff
@@ -0,0 +1,43 @@
+/* istanbul ignore file until EXP-1055 & EXP-1062 done */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Alert from "react-bootstrap/Alert";
+import Form from "react-bootstrap/Form";
+
+const FormApproveConfirm = ({
```

Comment:
>  it just opens RS yeah? No request or anything? 

There *is* a request, at least per Figma notes. That is, we should immediately trigger the push to Remote Settings that would otherwise get picked up by the background jobs:

![Capture](https://user-images.githubusercontent.com/21687/111683533-da508200-87e2-11eb-87c9-648ae2933512.PNG)

This is so that ideally the approval is ready to happen in Remote Settings just before opening it from the button. Otherwise, you get there early before the background task has run and there'd be nothing to approve yet.

Reply:
Ah okay I misunderstood what would be happening. Sounds good 👌